tools/test.py

```diff
@@ -142,9 +145,14 @@ def RunSingle(self, parallel, thread_id):
         return
       self.lock.acquire()
       if output.UnexpectedOutput():
-        self.failed.append(output)
-        if output.HasCrashed():
-          self.crashed += 1
+        if FLAKY in output.test.outcomes and self.flaky_tests_mode == "dontcare":
```

Comment:
Consistency nit: can you use a constant for the "dontcare" string like you did with FLAKY?